### ISSUE TITLE
VR view on small screen device

### ISSUE THREAD

**Issue by kpilyugin** [googlevr/cardboard#203]
Hi! We noticed that on device with small screen (iPod touch with 4'' screen) with default cardboard viewer profile VR view is not symmetric and distorted. We are using native iOS SDK, version 1.4.1.

<img width="752" alt="Screenshot 2021-02-04 at 12 43 59" src="https://user-images.githubusercontent.com/6966433/106888691-38b81a00-66e7-11eb-8c79-a0d8c5ca234f.png">

Previously we used our own implementation of stereo rendering which ignored "vertical alignment" parameter and always rendered vertically centered view. Testing showed that it looks fine on iPod in cardboard headset.

![IMG_7C8099B0ED02-1](https://user-images.githubusercontent.com/6966433/106889170-f0e5c280-66e7-11eb-8e4e-88e880217581.jpeg)

Could you please advise on the following:
- Are there minimum screen size requirements for SDK to work correctly and if there are, how they can be obtained?
- Would it make sense to force SDK to use center vertical alignment in this case?

**Comment by whizzkid** [comment]
This is the same as the original GoogleCardboard SDK handled it. 
Because the screen is physically smaller, but your eyes are still in the same position, it needs to render the image differently so that your eyes see the same while looking straight ahead, (without the user having to go slightly cross-eyed to properly see) but there's not enough physical screen to render pixels to.

![image](https://user-images.githubusercontent.com/323372/106901455-1084e700-66f8-11eb-8cd9-2292e3816516.png)

I've drawn a quick example.  the red lines shows sorta what your eyes see. 

So unless the image is warped while looking through the cardboard-headset. this is not a bug, but expected behaviour. Your previous implementation would've actually been the incorrect way to do it :)

if the image IS warped, then this is a bug.

**Comment by kpilyugin** [comment]
@whizzkid Yes that makes sense, my initial guess was the same, that SDK is using viewer profile correctly. But testing with real Cardboard headset showed the opposite: in current VR view images on small iPod are overlapping inside headset, and our old implementation with centered view looks good.

**Comment by whizzkid** [comment]
Ah ok, if the image is not correct while looking through the glasses then this seems to be an issue indeed.
